src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let getSum = 0;
+
+  const adder = (...args) => {
```

Comment:
this adder function takes only one argument, so I does not make any sense to use rest operator here